1 table.sql

```diff
@@ -0,0 +1,23 @@
+use InstrumentDB 
+go
+drop table if exists Instrument
+go 
+create table dbo.Instrument(
+InstrumentID int not null identity primary key ,
+CustomerFirstName varchar (50) not null constraint ck_Customer_First_Name_cannot_be_blank check (CustomerFirstName<>''),
+CustomerLastName varchar (50) not null constraint ck_Customer_Last_Name_cannot_be_blank check (CustomerLastName <>''),
+InstrumentName varchar(50) not null constraint ck_Instrument_Name_cannot_be_blank check(InstrumentName<>''),
+Category varchar(50) not null constraint ck_Instrument_Category_Cannot_be_blank check (Category<>''),
+InstrumentCode as concat (substring(CustomerFirstName,1,3),substring(Category,1,3)) persisted,
```

Comment:
Instrument Code is part Instrument name and part Category - not Customer First Name